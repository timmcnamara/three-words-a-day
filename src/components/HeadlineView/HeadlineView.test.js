import { render, cleanup } from "react-testing-library";
import React from "react";
import HeadlineView from ".";
import "jest-dom/extend-expect";

afterEach(cleanup);

describe("<HeadlineView />", () => {
  it("renders headline component", () => {
    const { asFragment } = render(<HeadlineView text="hello" />);

    // run a snapshot
    expect(asFragment()).toMatchSnapshot();
  });

  it("renders headline text", () => {
    const { getByText } = render(<HeadlineView title="hello" />);
    expect(getByText("hello")).toBeInTheDocument();
  });
});
