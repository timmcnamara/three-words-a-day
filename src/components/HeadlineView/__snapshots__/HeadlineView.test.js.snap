// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<HeadlineView /> renders headline component 1`] = `
<DocumentFragment>
  <h1
    class="sc-bdVaJa cFuNTo"
  />
</DocumentFragment>
`;
